ksmt-bitwuzla/src/test/kotlin/org/ksmt/solver/bitwuzla/ConverterTest.kt

```diff
@@ -12,6 +12,7 @@ import org.ksmt.solver.bitwuzla.bindings.FilePtrUtils
 import org.ksmt.solver.bitwuzla.bindings.Native
 import org.ksmt.sort.KBv1Sort
 import org.ksmt.sort.KSort
+import org.ksmt.utils.getValue
```

Comment:
Do we use it here? Do we have an example of usage?

Reply:
It is used in e.g. `val a by boolSort` (line 34).